### DISCUSSION TITLE
CUSA00918 Uncharted A Thief's End & CUSA08352 Uncharted The Lost Legacy

### DISCUSSION THREAD

**Discussion by drjackal99** [GoldHEN/GoldHEN_Cheat_Repository#50] [Cheat requests]
Please add CUSA00918 Uncharted A Thief's End [1.33] and CUSA08352 Uncharted The Lost Legacy [1.09]

i have tried code from PS4trainer : 
- CUSA00341 [1.33] works for CUSA00918 [1.33]
- CUSA09564 [1.08] works for CUSA08352 [1.09]

**Comment by kmeps4** [comment]
I need to test first and then upload them.. Thx
